2018-07-20 17:18:31,999 INFO [19609:MainThread] mopidy.__main__: Starting Mopidy 2.1.0
2018-07-20 17:18:32,009 INFO [19609:MainThread] mopidy.config: Loading config from builtin defaults
2018-07-20 17:18:32,010 INFO [19609:MainThread] mopidy.config: Loading config from /etc/mopidy/mopidy.conf
2018-07-20 17:18:32,010 INFO [19609:MainThread] mopidy.config: Loading config from command line options
2018-07-20 17:18:32,039 INFO [19609:MainThread] mopidy.__main__: Enabled extensions: spotify, mpd, http, stream, m3u, softwaremixer, file, local
2018-07-20 17:18:32,039 INFO [19609:MainThread] mopidy.__main__: Disabled extensions: none
2018-07-20 17:18:32,143 INFO [19609:MainThread] mopidy.commands: Starting Mopidy mixer: SoftwareMixer
2018-07-20 17:18:32,143 INFO [19609:MainThread] mopidy.commands: Starting Mopidy audio
2018-07-20 17:18:32,144 INFO [19609:MainThread] mopidy.commands: Starting Mopidy backends: StreamBackend, M3UBackend, FileBackend, LocalBackend, SpotifyBackend
2018-07-20 17:18:32,150 INFO [19609:Audio-2] mopidy.audio.actor: Audio output set to "tee name=t ! queue ! autoaudiosink t. ! queue ! udpsink port=5555"
2018-07-20 17:18:32,150 INFO [19609:MainThread] mopidy.local.json: No local library metadata cache found at /var/lib/mopidy/local/library.json.gz. Please run `mopidy local scan` to index your local music library. If you do not have a local music collection, you can disable the local backend to hide this message.
2018-07-20 17:18:32,150 INFO [19609:MainThread] mopidy.local.library: Loaded 0 local tracks using json
2018-07-20 17:18:32,154 INFO [19609:SpotifyBackend-7] mopidy.internal.path: Creating dir /var/cache/mopidy/spotify
2018-07-20 17:18:32,154 INFO [19609:SpotifyBackend-7] mopidy.internal.path: Creating dir /var/lib/mopidy/spotify
2018-07-20 17:18:32,155 INFO [19609:MainThread] mopidy.commands: Starting Mopidy core
2018-07-20 17:18:32,163 INFO [19609:MainThread] mopidy.commands: Starting Mopidy frontends: MpdFrontend, HttpFrontend
2018-07-20 17:18:32,164 INFO [19609:MainThread] mopidy.mpd.actor: MPD server running at [::ffff:127.0.0.1]:6600
2018-07-20 17:18:32,165 INFO [19609:HttpFrontend-14] mopidy.http.actor: HTTP server running at [::ffff:127.0.0.1]:6680
2018-07-20 17:18:32,166 INFO [19609:MainThread] mopidy.commands: Starting GLib mainloop
2018-07-20 17:18:32,279 ERROR [19609:SpotifyEventLoop] spotify.session: Spotify login error: <ErrorType.BAD_USERNAME_OR_PASSWORD: 6>
2018-07-20 17:18:32,280 INFO [19609:SpotifyEventLoop] spotify.session: Spotify logged out
2018-07-20 17:18:56,420 INFO [19609:MpdSession-15] mopidy.mpd.session: New MPD connection from [::ffff:127.0.0.1]:46766
2018-07-20 17:19:15,047 ERROR [19609:Core-9] mopidy.core.library: SpotifyBackend backend caused an exception.
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/mopidy/core/library.py", line 19, in _backend_error_handling
    yield
  File "/usr/lib/python2.7/dist-packages/mopidy/core/library.py", line 142, in get_distinct
    values = future.get()
  File "/usr/lib/python2.7/dist-packages/pykka/threading.py", line 52, in get
    compat.reraise(*self._data['exc_info'])
  File "/usr/lib/python2.7/dist-packages/pykka/compat.py", line 12, in reraise
    exec('raise tp, value, tb')
  File "/usr/lib/python2.7/dist-packages/pykka/actor.py", line 201, in _actor_loop
    response = self._handle_receive(message)
  File "/usr/lib/python2.7/dist-packages/pykka/actor.py", line 295, in _handle_receive
    return callee(*message['args'], **message['kwargs'])
  File "/usr/lib/python2.7/dist-packages/mopidy_spotify/library.py", line 26, in get_distinct
    field, query)
  File "/usr/lib/python2.7/dist-packages/mopidy_spotify/distinct.py", line 20, in get_distinct
    config, session, web_client, query)
  File "/usr/lib/python2.7/dist-packages/mopidy_spotify/distinct.py", line 45, in _get_distinct_artists
    for track in _get_playlist_tracks(config, session)
  File "/usr/lib/python2.7/dist-packages/mopidy_spotify/distinct.py", line 44, in <setcomp>
    artist.name
  File "/usr/lib/python2.7/dist-packages/mopidy_spotify/distinct.py", line 116, in _get_playlist_tracks
    for playlist in session.playlist_container:
TypeError: 'NoneType' object is not iterable
2018-07-20 17:19:15,318 ERROR [19609:Core-9] mopidy.core.library: SpotifyBackend backend caused an exception.
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/mopidy/core/library.py", line 19, in _backend_error_handling
    yield
  File "/usr/lib/python2.7/dist-packages/mopidy/core/library.py", line 142, in get_distinct
    values = future.get()
  File "/usr/lib/python2.7/dist-packages/pykka/threading.py", line 52, in get
    compat.reraise(*self._data['exc_info'])
  File "/usr/lib/python2.7/dist-packages/pykka/compat.py", line 12, in reraise
    exec('raise tp, value, tb')
  File "/usr/lib/python2.7/dist-packages/pykka/actor.py", line 201, in _actor_loop
    response = self._handle_receive(message)
  File "/usr/lib/python2.7/dist-packages/pykka/actor.py", line 295, in _handle_receive
    return callee(*message['args'], **message['kwargs'])
  File "/usr/lib/python2.7/dist-packages/mopidy_spotify/library.py", line 26, in get_distinct
    field, query)
  File "/usr/lib/python2.7/dist-packages/mopidy_spotify/distinct.py", line 20, in get_distinct
    config, session, web_client, query)
  File "/usr/lib/python2.7/dist-packages/mopidy_spotify/distinct.py", line 45, in _get_distinct_artists
    for track in _get_playlist_tracks(config, session)
  File "/usr/lib/python2.7/dist-packages/mopidy_spotify/distinct.py", line 44, in <setcomp>
    artist.name
  File "/usr/lib/python2.7/dist-packages/mopidy_spotify/distinct.py", line 116, in _get_playlist_tracks
    for playlist in session.playlist_container:
TypeError: 'NoneType' object is not iterable
2018-07-20 17:19:15,576 ERROR [19609:Core-9] mopidy.core.library: SpotifyBackend backend caused an exception.
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/mopidy/core/library.py", line 19, in _backend_error_handling
    yield
  File "/usr/lib/python2.7/dist-packages/mopidy/core/library.py", line 142, in get_distinct
    values = future.get()
  File "/usr/lib/python2.7/dist-packages/pykka/threading.py", line 52, in get
    compat.reraise(*self._data['exc_info'])
  File "/usr/lib/python2.7/dist-packages/pykka/compat.py", line 12, in reraise
    exec('raise tp, value, tb')
  File "/usr/lib/python2.7/dist-packages/pykka/actor.py", line 201, in _actor_loop
    response = self._handle_receive(message)
  File "/usr/lib/python2.7/dist-packages/pykka/actor.py", line 295, in _handle_receive
    return callee(*message['args'], **message['kwargs'])
  File "/usr/lib/python2.7/dist-packages/mopidy_spotify/library.py", line 26, in get_distinct
    field, query)
  File "/usr/lib/python2.7/dist-packages/mopidy_spotify/distinct.py", line 20, in get_distinct
    config, session, web_client, query)
  File "/usr/lib/python2.7/dist-packages/mopidy_spotify/distinct.py", line 45, in _get_distinct_artists
    for track in _get_playlist_tracks(config, session)
  File "/usr/lib/python2.7/dist-packages/mopidy_spotify/distinct.py", line 44, in <setcomp>
    artist.name
  File "/usr/lib/python2.7/dist-packages/mopidy_spotify/distinct.py", line 116, in _get_playlist_tracks
    for playlist in session.playlist_container:
TypeError: 'NoneType' object is not iterable
2018-07-20 17:19:15,834 ERROR [19609:Core-9] mopidy.core.library: SpotifyBackend backend caused an exception.
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/mopidy/core/library.py", line 19, in _backend_error_handling
    yield
  File "/usr/lib/python2.7/dist-packages/mopidy/core/library.py", line 142, in get_distinct
    values = future.get()
  File "/usr/lib/python2.7/dist-packages/pykka/threading.py", line 52, in get
    compat.reraise(*self._data['exc_info'])
  File "/usr/lib/python2.7/dist-packages/pykka/compat.py", line 12, in reraise
    exec('raise tp, value, tb')
  File "/usr/lib/python2.7/dist-packages/pykka/actor.py", line 201, in _actor_loop
    response = self._handle_receive(message)
  File "/usr/lib/python2.7/dist-packages/pykka/actor.py", line 295, in _handle_receive
    return callee(*message['args'], **message['kwargs'])
  File "/usr/lib/python2.7/dist-packages/mopidy_spotify/library.py", line 26, in get_distinct
    field, query)
  File "/usr/lib/python2.7/dist-packages/mopidy_spotify/distinct.py", line 20, in get_distinct
    config, session, web_client, query)
  File "/usr/lib/python2.7/dist-packages/mopidy_spotify/distinct.py", line 45, in _get_distinct_artists
    for track in _get_playlist_tracks(config, session)
  File "/usr/lib/python2.7/dist-packages/mopidy_spotify/distinct.py", line 44, in <setcomp>
    artist.name
  File "/usr/lib/python2.7/dist-packages/mopidy_spotify/distinct.py", line 116, in _get_playlist_tracks
    for playlist in session.playlist_container:
TypeError: 'NoneType' object is not iterable
2018-07-20 17:19:16,092 ERROR [19609:Core-9] mopidy.core.library: SpotifyBackend backend caused an exception.
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/mopidy/core/library.py", line 19, in _backend_error_handling
    yield
  File "/usr/lib/python2.7/dist-packages/mopidy/core/library.py", line 142, in get_distinct
    values = future.get()
  File "/usr/lib/python2.7/dist-packages/pykka/threading.py", line 52, in get
    compat.reraise(*self._data['exc_info'])
  File "/usr/lib/python2.7/dist-packages/pykka/compat.py", line 12, in reraise
    exec('raise tp, value, tb')
  File "/usr/lib/python2.7/dist-packages/pykka/actor.py", line 201, in _actor_loop
    response = self._handle_receive(message)
  File "/usr/lib/python2.7/dist-packages/pykka/actor.py", line 295, in _handle_receive
    return callee(*message['args'], **message['kwargs'])
  File "/usr/lib/python2.7/dist-packages/mopidy_spotify/library.py", line 26, in get_distinct
    field, query)
  File "/usr/lib/python2.7/dist-packages/mopidy_spotify/distinct.py", line 20, in get_distinct
    config, session, web_client, query)
  File "/usr/lib/python2.7/dist-packages/mopidy_spotify/distinct.py", line 45, in _get_distinct_artists
    for track in _get_playlist_tracks(config, session)
  File "/usr/lib/python2.7/dist-packages/mopidy_spotify/distinct.py", line 44, in <setcomp>
    artist.name
  File "/usr/lib/python2.7/dist-packages/mopidy_spotify/distinct.py", line 116, in _get_playlist_tracks
    for playlist in session.playlist_container:
TypeError: 'NoneType' object is not iterable
2018-07-20 17:24:04,875 INFO [26630:MainThread] mopidy.__main__: Starting Mopidy 2.1.0
2018-07-20 17:24:04,881 INFO [26630:MainThread] mopidy.config: Loading config from builtin defaults
2018-07-20 17:24:04,881 INFO [26630:MainThread] mopidy.config: Loading config from /etc/mopidy/mopidy.conf
2018-07-20 17:24:04,882 INFO [26630:MainThread] mopidy.config: Loading config from command line options
2018-07-20 17:24:04,901 INFO [26630:MainThread] mopidy.__main__: Enabled extensions: spotify, mpd, http, stream, m3u, softwaremixer, file, local
2018-07-20 17:24:04,901 INFO [26630:MainThread] mopidy.__main__: Disabled extensions: none
2018-07-20 17:24:04,963 INFO [26630:MainThread] mopidy.commands: Starting Mopidy mixer: SoftwareMixer
2018-07-20 17:24:04,964 INFO [26630:MainThread] mopidy.commands: Starting Mopidy audio
2018-07-20 17:24:04,965 INFO [26630:MainThread] mopidy.commands: Starting Mopidy backends: StreamBackend, M3UBackend, FileBackend, LocalBackend, SpotifyBackend
2018-07-20 17:24:04,970 INFO [26630:MainThread] mopidy.local.json: No local library metadata cache found at /var/lib/mopidy/local/library.json.gz. Please run `mopidy local scan` to index your local music library. If you do not have a local music collection, you can disable the local backend to hide this message.
2018-07-20 17:24:04,970 INFO [26630:MainThread] mopidy.local.library: Loaded 0 local tracks using json
2018-07-20 17:24:04,970 INFO [26630:Audio-2] mopidy.audio.actor: Audio output set to "tee name=t ! queue ! autoaudiosink t. ! queue ! udpsink port=5555"
2018-07-20 17:24:04,975 INFO [26630:MainThread] mopidy.commands: Starting Mopidy core
2018-07-20 17:24:04,983 INFO [26630:MainThread] mopidy.commands: Starting Mopidy frontends: MpdFrontend, HttpFrontend
2018-07-20 17:24:04,984 INFO [26630:MainThread] mopidy.mpd.actor: MPD server running at [::ffff:127.0.0.1]:6600
2018-07-20 17:24:04,985 INFO [26630:HttpFrontend-13] mopidy.http.actor: HTTP server running at [::ffff:127.0.0.1]:6680
2018-07-20 17:24:04,985 INFO [26630:MainThread] mopidy.commands: Starting GLib mainloop
2018-07-20 17:24:05,090 ERROR [26630:SpotifyEventLoop] spotify.session: Spotify login error: <ErrorType.BAD_USERNAME_OR_PASSWORD: 6>
2018-07-20 17:24:05,092 INFO [26630:SpotifyEventLoop] spotify.session: Spotify logged out
2018-07-20 17:24:39,070 INFO [26630:MainThread] mopidy.commands: GLib mainloop got SIGTERM. Exiting...
2018-07-20 17:24:39,070 INFO [26630:MainThread] mopidy.commands: Stopping Mopidy frontends
2018-07-20 17:24:39,072 INFO [26630:MainThread] mopidy.commands: Stopping Mopidy core
2018-07-20 17:24:39,073 INFO [26630:MainThread] mopidy.commands: Stopping Mopidy backends
2018-07-20 17:24:39,074 INFO [26630:MainThread] mopidy.commands: Stopping Mopidy audio
2018-07-20 17:24:39,075 INFO [26630:MainThread] mopidy.commands: Stopping Mopidy mixer
2018-07-20 17:26:09,316 INFO [29313:MainThread] mopidy.__main__: Starting Mopidy 2.1.0
2018-07-20 17:26:09,322 INFO [29313:MainThread] mopidy.config: Loading config from builtin defaults
2018-07-20 17:26:09,323 INFO [29313:MainThread] mopidy.config: Loading config from /etc/mopidy/mopidy.conf
2018-07-20 17:26:09,323 INFO [29313:MainThread] mopidy.config: Loading config from command line options
2018-07-20 17:26:09,342 INFO [29313:MainThread] mopidy.__main__: Enabled extensions: spotify, mpd, http, stream, m3u, softwaremixer, file, local
2018-07-20 17:26:09,342 INFO [29313:MainThread] mopidy.__main__: Disabled extensions: none
2018-07-20 17:26:09,408 INFO [29313:MainThread] mopidy.commands: Starting Mopidy mixer: SoftwareMixer
2018-07-20 17:26:09,409 INFO [29313:MainThread] mopidy.commands: Starting Mopidy audio
2018-07-20 17:26:09,410 INFO [29313:MainThread] mopidy.commands: Starting Mopidy backends: StreamBackend, M3UBackend, FileBackend, LocalBackend, SpotifyBackend
2018-07-20 17:26:09,415 INFO [29313:MainThread] mopidy.local.json: No local library metadata cache found at /var/lib/mopidy/local/library.json.gz. Please run `mopidy local scan` to index your local music library. If you do not have a local music collection, you can disable the local backend to hide this message.
2018-07-20 17:26:09,415 INFO [29313:MainThread] mopidy.local.library: Loaded 0 local tracks using json
2018-07-20 17:26:09,416 INFO [29313:Audio-2] mopidy.audio.actor: Audio output set to "tee name=t ! queue ! autoaudiosink t. ! queue ! udpsink port=5555"
2018-07-20 17:26:09,420 INFO [29313:MainThread] mopidy.commands: Starting Mopidy core
2018-07-20 17:26:09,428 INFO [29313:MainThread] mopidy.commands: Starting Mopidy frontends: MpdFrontend, HttpFrontend
2018-07-20 17:26:09,429 INFO [29313:MainThread] mopidy.mpd.actor: MPD server running at [::ffff:127.0.0.1]:6600
2018-07-20 17:26:09,430 INFO [29313:HttpFrontend-13] mopidy.http.actor: HTTP server running at [::ffff:127.0.0.1]:6680
2018-07-20 17:26:09,430 INFO [29313:MainThread] mopidy.commands: Starting GLib mainloop
2018-07-20 17:26:09,537 ERROR [29313:SpotifyEventLoop] spotify.session: Spotify login error: <ErrorType.BAD_USERNAME_OR_PASSWORD: 6>
2018-07-20 17:26:09,539 INFO [29313:SpotifyEventLoop] spotify.session: Spotify logged out
2018-07-20 17:27:28,726 INFO [29313:MainThread] mopidy.commands: GLib mainloop got SIGTERM. Exiting...
2018-07-20 17:27:28,727 INFO [29313:MainThread] mopidy.commands: Stopping Mopidy frontends
2018-07-20 17:27:28,734 INFO [29313:MainThread] mopidy.commands: Stopping Mopidy core
2018-07-20 17:27:28,735 INFO [29313:MainThread] mopidy.commands: Stopping Mopidy backends
2018-07-20 17:27:28,740 INFO [29313:MainThread] mopidy.commands: Stopping Mopidy audio
2018-07-20 17:27:28,741 INFO [29313:MainThread] mopidy.commands: Stopping Mopidy mixer
